ui-react/config-overrides.js

```diff
@@ -0,0 +1,3 @@
+const { useBabelRc, override } = require("customize-cra");
```

Comment:
For minimizing bundle size and cleaner material-ui imports.
https://material-ui.com/guides/minimizing-bundle-size/

Reply:
Imports should be updated after this?

Reply:
Not necessarily. I believe it gives us the option to use the shorter import version and take advantage of tree shaking. At least I didn't see any errors.